app/main.py

```diff
@@ -1,8 +1,41 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Use indentation like this to increase the readability of your code. The same in other similar cases.


```suggestion
    def __init__(
        self,
        comfort_class: int,
        clean_mark: int,
        brand: str
) -> None:
```